examples/CORE/biolatency.bpf.c

```diff
@@ -0,0 +1,175 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+#include <vmlinux.h>
```

Comment:
Where is this file supposed to come from?

Reply:
https://github.com/iovisor/bcc/tree/master/libbpf-tools/x86

Reply:
What should an end user do when all they have is a binary or the source of this repo?